extra/release_tool.py

```diff
@@ -370,6 +155,9 @@ def yml_components(self):
             comp.type = "yml"
         return comps
 
+    def is_release_component(self):
```

Comment:
It works because so far it is only used once in which case we have a custom map.

But by design this method should call `_initialize_component_maps()`

Reply:
Oh, nice catch! Indeed, I will fix that!

Reply:
One more function was affected as well. Fixed both.

And the test runner cancelled my old build beautifully!

Reply:
Nice! Thanks for reporting.